Scrypt's build system does not expose a reusable version of crypto_scrypt(). To
work around this, the following patch adds a library that achieves this.

diff --git Makefile.am Makefile.am
index f037aa6..616c01d 100644
--- Makefile.am
+++ Makefile.am
@@ -69,6 +69,22 @@ noinst_LIBRARIES+=	libscrypt_sse2.a
 libscrypt_sse2_a_SOURCES=	lib/crypto/crypto_scrypt_smix_sse2.c
 libscrypt_sse2_a_CFLAGS=`. ./cpusupport-config.h; echo $${CFLAGS_X86_SSE2}`
 
+noinst_LIBRARIES+= libcrypto_scrypt.a
+libcrypto_scrypt_a_SOURCES= \
+			libcperciva/alg/sha256.c			\
+			libcperciva/alg/sha256.h			\
+			libcperciva/cpusupport/cpusupport_x86_sse2.c	\
+			libcperciva/util/insecure_memzero.c		\
+			libcperciva/util/insecure_memzero.h		\
+			libcperciva/util/warnp.c			\
+			libcperciva/util/warnp.h			\
+			lib/crypto/crypto_scrypt.c			\
+			lib/crypto/crypto_scrypt.h			\
+			lib/crypto/crypto_scrypt_smix.c			\
+			lib/crypto/crypto_scrypt_smix.h			\
+			lib/crypto/crypto_scrypt_smix_sse2.c			\
+			lib/crypto/crypto_scrypt_smix_sse2.h
+
 EXTRA_DIST = \
 	COPYRIGHT						\
 	FORMAT							\
